<launch>
	<!-- run the autonomous_landing.cpp -->
	<node pkg="prometheus_mission" type="autonomous_landing_modified" name="autonomous_landing_modified" output="screen" launch-prefix="gnome-terminal --">
		<!-- 仿真模式 - 区别在于是否自动切换offboard模式 -->
		<param name="sim_mode" value="true" />
		<!-- 悬停模式 - 仅用于观察检测结果 -->
		<param name="hold_mode" value="false" />
		<!-- 是否使用降落板绝对高度 -->
		<param name="use_pad_height" value="true" />
		<param name="pad_height" value="0.99" />
		<!-- 起始点 -->
		<param name="start_point_x" value="-4.0" />
		<param name="start_point_y" value="-2.0" />
		<param name="start_point_z" value="4.5" />
		<!-- 相机安装偏差 -->
		<param name="camera_offset_x" value="0.0" />
		<param name="camera_offset_y" value="0.0" />
		<param name="camera_offset_z" value="-0.1" />
		<!-- 追踪控制参数 -->
		<param name="kpx_land" value="0.5" />
		<param name="kpy_land" value="0.5" />
		<param name="kpz_land" value="0.12" />
		<param name="arm_height_to_ground" value="0.5" />
		<param name="arm_distance_to_pad" value="0.6" />
		<!-- 目标是否移动及其速度 -->
		<param name="moving_target" value="true" />
		<param name="target_vel_x" value="1.1" />
		<param name="target_vel_y" value="0.0" />
	</node>

</launch>

